src/styles.scss

```diff
@@ -1,3 +1,18 @@
+@import '~@angular/material/theming';
+@include mat-core();
+
+$my-app-primary: mat-palette($mat-indigo);
+$my-app-accent: mat-palette($mat-pink, A200, A100, A400);
+
+$my-app-theme: mat-light-theme((
+  color: (
+    primary: $my-app-primary,
+    accent: $my-app-accent,
+  )
+));
+
+@include angular-material-theme($my-app-theme);
+
```

Comment:
Why do we need this in the global styles?

Reply:
If we remove the first two lines, the layout of the Material dropdown menus will break. I will delete the rest of the styles.

Reply:
Why? Aren't you already including the core angular material styles as part of the sample?